jobs/cache_refresh/pyproject.toml

```diff
@@ -0,0 +1,44 @@
+[tool.poetry]
+authors = ["Andrea Soria <andrea@huggingface.co>"]
+description = "Refresh outdated cache entries based on worker version"
```

Comment:
```suggestion
description = "Refresh outdated cache entries based on job runner version"
```

Reply:
I think you didn't apply this suggestion. "job runner" would be more accurate than "worker"

Reply:
Yeah sorry I forgot to apply it